app/components/UI/TransactionReview/index.js

```diff
@@ -367,7 +367,7 @@ class TransactionReview extends PureComponent {
 							onCancelPress={this.props.onCancel}
 							onConfirmPress={this.props.onConfirm}
 							confirmed={transactionConfirmed}
-							confirmDisabled={transactionConfirmed || error || isAnimating}
+							confirmDisabled={transactionConfirmed || !!error || isAnimating}
```

Comment:
rather than using `!!` here... which I am not a fan of... does it make sense to do this instead?
```suggestion
							confirmDisabled={transactionConfirmed || Boolean(error) || isAnimating}
```
`Boolean(undefined)` returns false... so I believe it accomplishes the same while being a bit more readable.